src/component/misc/StringListEdit.scss

```diff
@@ -1,3 +1,7 @@
-.no-shadow-box {
-  box-shadow: none !important;
+.no-bottom-margin {
+  margin-bottom: 0rem !important;
+}
+
+.form-group-bottom-margin {
```

Comment:
Doesn't regular form-group cover the bottom margin?

Reply:
it does, but there is a problem with the textual hint when being part of the group (on large resolutions the hint is layouted on the same row as the input/button group).